mapboxgl/viz.py

```diff
@@ -187,3 +187,90 @@ def create_html(self):
             options.update(labelProperty='{' + self.label_property + '}')
 
         return templates.format('graduated_circle', **options)
+
+
+class HeatmapViz(object):
+    """Create a heatmap viz"""
+
+    def __init__(self,
+                 data,
+                 access_token=None,
+                 center=(0, 0),
+                 weight_property=None,
+                 weight_stops=None,
+                 color_stops=None,
+                 radius_stops=None,
+                 div_id='map',
+                 height='500px',
+                 style_url="mapbox://styles/mapbox/light-v9?optimize=true",
```

Comment:
neat trick, style optimized vector tiles! https://blog.mapbox.com/style-optimized-vector-tiles-39868da81275

Reply:
On by default to speed up base map rendering and network tile requests.